examples/sdk/basic-typescript-example/src/integration.ts

```diff
@@ -0,0 +1,168 @@
+  );
+};
+
+// Initialize editor
+SDK.loadEditor();
+
+// Load document
+const loadDocument = async () => {
+  await SDK.document.load(STUDIO_DOCUMENT);
+  zoomToPage();
+};
+loadDocument();
+
+// Tool selection and change available on window
+const useSelectTool = () => {
+  SDK.tool.setSelect();
+};
+
+const useHandTool = () => {
+  SDK.tool.setHand();
+};
+
+const useZoomTool = () => {
+  SDK.tool.setZoom();
+};
+
+const onToolChanged = (tool: ToolType) => {
+  if (tool) {
+    const toolLabel = document.getElementById("toolLabel");
+    toolLabel.textContent = "Selected tool: " + tool;
+  }
+};
+
+// Play animation
+const playAnimation = async () => {
+  SDK.animation.play();
+};
+
+// Functions on frame selection
+const onFrameContentChange = (selectedFrameContent: Frame) => {
+  if (selectedFrameContent) {
+    const frameTitleInput = document.getElementById("frameTitle");
+    frameTitleInput.setAttribute("value", selectedFrameContent.name);
+
+    const frameTypeInput = document.getElementById("frameType");
+    frameTypeInput.setAttribute("value", selectedFrameContent.type);
+  }
+};
+const onFrameLayoutChange = (selectedFrameLayout: FrameLayoutType) => {
+  if (selectedFrameLayout) {
+    const frameXInput = document.getElementById("frameX");
+    frameXInput.setAttribute("value", String(selectedFrameLayout.x.value));
+
+    const frameYInput = document.getElementById("frameY");
+    frameYInput.setAttribute("value", String(selectedFrameLayout.y.value));
+
+    const frameWidthInput = document.getElementById("frameWidth");
+    frameWidthInput.setAttribute(
+      "value",
+      String(selectedFrameLayout.width.value)
+    );
+
+    const frameHeightInput = document.getElementById("frameHeight");
+    frameHeightInput.setAttribute(
+      "value",
+      String(selectedFrameLayout.height.value)
+    );
+  }
+};
+
+// Select a layout
+const onLayoutClick = (id: string) => {
+  SDK.layout.select(id);
+};
+// Function on when a layout has been changed
+const onLayoutsChanged = (layouts: LayoutListItemType[]) => {
+  if (layouts && layouts.length) {
+    const listContainer = document.getElementById("layoutList");
+    // Empty list on rerender
+    listContainer.innerHTML = "";
+
+    // loop all layouts and render them + add dynamic onClick handler
+    layouts.map((layout) => {
+      const item = document.createElement("li");
+      item.setAttribute("class", "layout-item");
+      if (layout.id === SELECTED_LAYOUT_ID) {
+        //@ts-ignore
+        item.classList = `${item.classList} selected`;
+      }
+      item.setAttribute("id", `layout ${layout.id} ${layout.name}`);
+      item.onclick = () => onLayoutClick(layout.id);
+      const itemText = document.createTextNode(layout.name);
+      item.appendChild(itemText);
+      listContainer.appendChild(item);
+    });
+  }
+};
+
+const onSelectedLayoutChanged = (id: string) => {
+  console.log(`Selected layout: ${id}`);
+  SELECTED_LAYOUT_ID = id;
+};
+
+export default {
```

Comment:
I'm not sure but shouldn't default import be only one?

Reply:
I don't think it's persé wrong, otherwise I would make the object the same and export it, that feels like a choice no?